lib/index.js

```diff
@@ -15,7 +15,7 @@ module.exports = (pluginConfig, assetManager, { watcher, browsers, compact }) =>
 function buildConfig(pluginConfig, assetManager, { watcher, browsers, compact }) {
 	return pluginConfig.map(bundleConfig => {
 		let inputFileName = assetManager.resolvePath(bundleConfig.source, {
-			enforceRelative: true
+			enforceRelative: false
```

Comment:
Might as well omit this `options`(!) object entirely?

(ditto for faucet-static btw)